### PR TITLE
Fix typos in documentation

### PR DESCRIPTION
**Pull request by MarionLeHerisson**
Fixed a few typos in the ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -261,8 +260,8 @@ If you add a new Fixer, please provide sources and references about the typograp
 Compatibility & OS support restrictions
 =======================================
 
-- Windows XP : Thin No-Break Space can't be used, all other spaces are ignored but they do not look bad (normal space).
-- Mac OS Snow Leopard : no espaces fixes, demi-fixes, cadratin et demi-cadratin but does not look bad (normal space).
+- Windows XP : Thin No-Break Space can't be used, all other spaces are ignored, but they do not look bad (normal space).
+- Mac OS Snow Leopard : no espaces fixes, demi-fixes, cadratin and half-cadratin but doesn't look bad (normal space).
```
Comment (pyrech) [inline README.md:264]: I'm not sure about "half-cadratin", Google doesn't return much results for it

**Review by pyrech — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (damienalexandre) [inline README.md:264]: The proper translation for "demi-cadratin" is en-dash. But the other type of spaces are not translated here, so maybe it should not be changed.

**Review by damienalexandre — APPROVED** [review]
Awesome, thanks a lot

File: README.md (hunk 1 above)
Comment (MarionLeHerisson) [reply README.md:264]: Just checked on Wikipedia : the French article about "cadratin" is translated by "em", I'll fix this :) 
https://en.wikipedia.org/wiki/Em_(typography)

Comment (MarionLeHerisson) [reply README.md:264]: I'm indeed not sure about how to translate "espaces fixes" and "demi-fixes" :grimacing: "fixed spaces" and "thin space" ?

Comment (damienalexandre) [reply README.md:264]: think this was about no-break space. so it would be "no no-break space, half no-break space".

Comment (MarionLeHerisson) [reply README.md]: Here you go :smile:

File: README.md [hunk 2]
```diff
@@ -220,33 +220,32 @@ Then just give JoliTypo their fully qualified name, or even instance:
...
 ```
 
 Add your own Fixer / Contribute a Fixer
 =======================================
 
-- Write test;
+- Write tests;
 - A Fixer is run on a piece of text, no HTML to deal with;
 - Implement `JoliTypo\FixerInterface`;
 - Pull request;
```
Comment (damienalexandre) [inline README.md:248]:
```suggestion
- Send your Pull request.
```

**Comment by damienalexandre** [comment]
Thanks @MarionLeHerisson :pray:

**Pull request merged**